docs/src/examples/mpg_fr.adoc

```diff
@@ -13,66 +13,147 @@ trois incréments de pas: 0.1, 0.01, 0.001.
 
 Dans le fichier _custom.hal_ ou dans un fichier jog.hal, ajouter
 ce qui suit en vérifiant bien que les composants _mux4_ ou _encoder_ ne
-soient pas déjà utilisés. Si c'était le cas il faudrait en augmenter le nombre 
+soient pas déjà utilisés. Si c'était le cas il faudrait en augmenter le nombre
 en incrémentant la valeur du _count_ de la commande _loadrt_. Ajuster également
 le numéro de référence.
 Les composants mux4 et encoder sont décrits dans le manuel de HAL et dans les
 man pages.
 
+See the <<sec:hal-section,HAL Ini Section>> of the manual for more
+information on adding a hal file.
+
+Jog management pins are provided for each joint and all coordinate
+letters.  This example uses the axis jog pins for jogging in world
+mode.  Machines with non-identity kinematics may need use additional
+connections for jogging in joint mode.
+
+Manivelle de jog
 
-.Manivelle de jog
 ----
+# Jog Pendant
 loadrt encoder num_chan=1
 loadrt mux4 count=1
 addf encoder.capture-position servo-thread
 addf encoder.update-counters base-thread
 addf mux4.0 servo-thread
 
-# Mode position 
-# Chaque cran de manivelle provoque un pas calibré,
-# la durée du mouvement total peut dépasser la durée de rotation de la manivelle.
-# C'est le mode par défaut.
+# If your MPG outputs a quadrature signal per click set x4 to 1
+# If your MPG puts out 1 pulse per click set x4 to 0
+setp encoder.0.x4-mode 0
+
+# For velocity mode, set to 1
+# In velocity mode the axis stops when the dial is stopped
+# even if that means the commanded motion is not completed,
+# For position mode (the default), set to 0
+# In position mode the axis will move exactly jog-scale
+# units for each count, regardless of how long that might take,
+setp axis.x.jog-vel-mode 0
+setp axis.y.jog-vel-mode 0
+setp axis.z.jog-vel-mode 0
+
+# This sets the scale that will be used based on the input to the mux4
+setp mux4.0.in0 0.1
+setp mux4.0.in1 0.01
+setp mux4.0.in2 0.001
 
-setp axis.N.jog-vel-mode 0
+# The inputs to the mux4 component
+net scale1 mux4.0.sel0 <= parport.1.pin-09-in
+net scale2 mux4.0.sel1 <= parport.1.pin-10-in
 
-# Mode vitesse
-# L'axe s'arrête quand la manivelle s'arrête, même si la pas de jog est incomplet.
-# Décommenter la ligne suivante pour obtenir ce mode de fonctionnement,
-# et commenter le mode position.
```

Comment:
Same as above

Reply:
And I agree again. I remember these paragraphs. It somehow did not fit and at some point what does not get fixed gets axed. With Petter burning out on po4a the focus is on getting as much into .po as quickly as possible and the documentation has too many difficulties to care for such imperfections.

Reply:
They were actually present in english source.
Just explicited the fact that machine may keep on moving after dial is stopped as it seem a worth notice ;)